app/backend/registries/views.py

```diff
@@ -81,9 +81,10 @@ class AuditCreateMixin(CreateModelMixin):
     """
 
     def perform_create(self, serializer):
-        serializer.validated_data['create_user'] = (self.request.user.profile.username or
-                                                    self.request.user.profile.name or
-                                                    self.request.user.get_username())
+        if self.request.user.profile.username is None:
+            raise exceptions.ValidationError(('Username must be set.'))
+
+        serializer.validated_data['create_user'] = (self.request.user.profile.username)
```

Comment:
Does it need to be in brackets?
```suggestion
        serializer.validated_data['create_user'] = self.request.user.profile.username
```